index.bs

```diff
@@ -4629,14 +4886,33 @@ partial interface MLGraphBuilder {
         - *SumSquare*: Compute the sum of the square of all the input values along the axes.
 </div>
 
+<details open algorithm>
+  <summary>
+    To <dfn for="MLGraphBuilder">calculate reduction output sizes</dfn>, given a [=/list=] of unsigned integers |inputShape|, a optional [=/list=] of unsigned integers |axes|, and [=/boolean=] |keepDimensions|, perform the following steps. They return a new [=/list=] of unsigned integers, or failure.
+  </summary>
+  <div class=algorithm-steps>
+    1. Let |inputSize| be |inputShape|'s [=list/size=].
```

Comment:
Would |inputRank| be more consistent within the spec?

Reply:
If I search the spec I see "size be" turns up 11 times, "rank be" 0 times. 

(Rank might be better generally, but seems like that's a separate issue?)

Reply:
Good point. `inputSize` is quite ambiguous alone because many ML libraries have a `Size()` operator which (for better or worse, and probably should have been called `ElementCount()` instead) returns the total *number of elements*, not the count of dimensions. So, `inputRank` or `inputShapeSize` or `inputDimensionCount` would all clearer names (with `inputRank` being the most concise), but deferring that to a separate CR that replaces the other 11 occurrences would be fine too.

Reply:
@fdwr a W3C parlance nit: TL;DR: s/CR/PR

Confusingly CR has a [domain-specific meaning](https://www.w3.org/2023/Process-20231103/#RecsCR) in the W3C context. It does not refer to the usual generic term Change Request that is familiar from change management process. A W3C spec junkie might interpret CR in this context to mean Candidate Recommendation. To avoid this name collision, we've used PR to refer to a Pull Request.

To not make things too simple, each successful merge at this stage of the spec evolution will in fact [produce](https://www.w3.org/standards/history/webnn/) a new [Candidate Recommendation Draft](https://www.w3.org/2023/Process-20231103/#publishing-crud) of the spec so basically what you're saying above is accurate.

(I don't want to mention also [PR is overloaded](https://www.w3.org/2023/Process-20231103/#RecsPR), but let's just pretend PR means Pull Request and feel good :-))

Reply:
🧐 Then under webnn/*, I'll just always spell it out (CR -> code review / change request, PR ->  pull request).

Reply:
Captured as #588 so resolving this conversation.